src/Storefront/Theme/Command/ThemePrepareIconsCommand.php

```diff
@@ -149,7 +154,8 @@ protected function execute(InputInterface $input, OutputInterface $output): int
                 $this->removeStyles($svg->getDocument());
             }
 
-            file_put_contents($path . 'processed/' . basename($file), $svg->toXMLString(false));
+            $fs->touch($path . 'processed/' . $file->getBasename());
+            $fs->appendToFile($path . 'processed/' . $file->getBasename(), $svg->toXMLString(false));
```

Comment:
```suggestion
            $fs->dumpFile($path . 'processed/' . $file->getBasename(), $svg->toXMLString(false));
```

why not using `dumpFile`?

Reply:
True 🤔 dumpFile is more what is used in our other commands